### DISCUSSION TITLE
> i have same issue when i use livewire in contact form while typing form automatically send GET request and return with exception The GET method is not supported for this route. Supported methods: POST. i can not figure out whats happens ... ![Untitled](https://user-images.githubusercontent.com/33140498/100799228-6950ae80-342d-11eb-864a-7d7d042ab6b7.png)

### DISCUSSION THREAD

**Discussion by Yondalne** [livewire/livewire#8649] [1. Help]
> i have same issue when i use livewire in contact form while typing form automatically send GET request and return with exception The GET method is not supported for this route. Supported methods: POST. i can not figure out whats happens ... ![Untitled](https://user-images.githubusercontent.com/33140498/100799228-6950ae80-342d-11eb-864a-7d7d042ab6b7.png)

Hi,
Have the same issue there, somehow the first time I'm going to my login page and try to enter some information I have this error.
But after a reload, it goes away but it is still happening everytime I'm trying with a new browser or if i completely close the tab. (it doesn't happen with php artisan serve)
I'm using laravel 10.10 and livewire 3.4 
Any idea of how to solve it ?

_Originally posted by @Yondalne in https://github.com/livewire/livewire/issues/1779#issuecomment-2212396200_

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Reply by Yondalne** [reply]
Okay, thanks
